WeakAurasTemplates/TriggerTemplates.lua

```diff
@@ -962,7 +1007,16 @@ function WeakAuras.CreateTemplateView(frame)
       end
       button:SetFullWidth(true);
       button:SetClick(function()
-        if (newView.existingAura) then
+        if newView.batchStep then
+          for x, b in pairs(subTypesButtons) do
```

Comment:
x is a terrible variable name.

Reply:
Renamed both for `for index, subTypesButton in`
Btw, newView.batchStep was called like that because at the time i did it, i though i would make more screens after the selection is done (for group creation), and was intended to be of the type number later.